### PR TITLE
Taxon translation uidx by locale

### PR DESCRIPTION
**Pull request by gonzalovilaseca**
Permalink unique index should be by permalink and locale, as it's possible to have two differente locales with the same permalink.

### CODE REVIEW

File: src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/TaxonTranslation.orm.xml [hunk 1]
```diff
@@ -23,11 +23,11 @@
...
-        <field name="slug" column="slug" type="string" unique="true">
-            <gedmo:slug fields="name" unique="true" />
+        <field name="slug" column="slug" type="string" unique="false">
+            <gedmo:slug fields="name" unique="false" />
         </field>
-        <field name="permalink" column="permalink" type="string" unique="true">
-            <gedmo:slug fields="name" unique="true" >
+        <field name="permalink" column="permalink" type="string" unique="false">
+            <gedmo:slug fields="name" unique="false" >
                 <!--TODO custom handler for translations-->
```
Comment (stloyd) [inline src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/TaxonTranslation.orm.xml]: I guess that this TODO can be removed after this PR, no?

Comment (gonzalovilaseca) [inline src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/TaxonTranslation.orm.xml]: I have to look further into this, as I don't really know what the handlers do, will look at it and update this PR with the solution.

Comment (gonzalovilaseca) [inline src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/TaxonTranslation.orm.xml]: @stloyd The TODO is fixed here  #2431

**Comment by gonzalovilaseca** [comment]
Please close this PR without merging as the changes are going to be included in #2431

**Pull request closed without merging**